### PR TITLE
At least make the fake JS some what real JS 🤗

### PR DESCRIPTION
**Pull request by craigmr**
This is breaking my webpack. Making pull request for others to review.

### CODE REVIEW

File: Explorer/MoreExploration.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-function Explore()
+class Explore()
```
Comment (mcphersonwhite) [inline Explorer/MoreExploration.js:1]: so we are going to use a class huh?

File: Explorer/MoreExploration.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
 {
-	this.Explore();
-}
\ No newline at end of file
+   iamARealFunction() {
```
Comment (mcphersonwhite) [inline Explorer/MoreExploration.js]: casing in function name is a bit off. should be iAmARealFunction. But that is nitpicky.

**Review by mcphersonwhite — APPROVED** [review]
approved

File: Explorer/MoreExploration.js (hunk 1 above)
Comment (craigmr) [reply Explorer/MoreExploration.js:1]: Just because they drive Baker crazy.

**Pull request merged**